dojo/templates/dojo/custom_html_report_finding_list.html

```diff
@@ -96,14 +108,16 @@ <h5>
         {% if finding.cvssv3 %}
             <h6>CVSS v3</h6>
             <pre>{{ finding.cvssv3|markdown_render }}</pre>
-            {% endif %}
+        {% endif %}
 
-        <h6>Description</h6>
-        <pre>{{ finding.description|markdown_render }}</pre>
+        {% if finding.description %}
+            <h6>Description</h6>
+            <pre>{{ finding.description|markdown_render }}</pre>
+        {% endif %}
 
-        <h6>Mitigation</h6>
-        <pre>{{ finding.mitigation|markdown_render }}</pre>
         {% if finding.get_report_requests %}
+            <h6>Mitigation</h6>
```

Comment:
Putting the mitigation in this if block will only allow the mitigation section to appear in the report if the finding has request/response pairs. That means it will only be available for dynamic tools.